images/macos/scripts/build/configure-machine.sh

```diff
@@ -76,11 +73,6 @@ EOF
     sleep 10
 done
 
-echo "Getting terminal windows"
```

Comment:
Is this code deleted deliberately?

Reply:
Yes..since we use "expect" command now it is no longer opening a terminal to run the code for enabling automationmodetool. Tested if there are any other terminal sessions open and dont see any.